index.html

```diff
@@ -430,11 +430,11 @@ <h3 id="cdn-paas"><a href="#cdn-paas">CDN &amp; PaaS performance</a></h3>
             <td>Instart Logic</td>
             <td class="ok">yes</td>
             <td class="ok">yes</td>
-            <td class="alert">no</td>
+            <td class="ok">yes</td>
             <td class="warn">configurable (static)</td>
+            <td class="warn">no</td>
```

Comment:
This column is for ALPN. it should be "yes":
```
$ openssl s_client -connect www.instartlogic.com:443 -alpn h2 </dev/null 2>/dev/null|grep ALPN
...
ALPN protocol: h2
```

Also H2 is supported and was announced here: https://www.instartlogic.com/blog/upgrading-http2

@wesleyhales can you add that link to the last column?